2022-04-22 17:46:25 Error: [MissingViewException] View file "Users\login.ctp" is missing.
Exception Attributes: array (
  'file' => 'Users\\login.ctp',
)
Request URL: /mantenedor-usuarios/users/login
Stack Trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(470): View->_getViewFileName('Users\\login')
#1 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Controller\Controller.php(968): View->render(NULL, NULL)
#2 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(200): Controller->render()
#3 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2022-04-22 17:47:28 Error: [MissingViewException] View file "Users\add.ctp" is missing.
Exception Attributes: array (
  'file' => 'Users\\add.ctp',
)
Request URL: /mantenedor-usuarios/users/add
Stack Trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(470): View->_getViewFileName('Users\\add')
#1 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Controller\Controller.php(968): View->render(NULL, NULL)
#2 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(200): Controller->render()
#3 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2022-04-22 17:48:06 Error: [MissingViewException] View file "Users\index.ctp" is missing.
Exception Attributes: array (
  'file' => 'Users\\index.ctp',
)
Request URL: /mantenedor-usuarios/users
Stack Trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(470): View->_getViewFileName('Users\\index')
#1 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Controller\Controller.php(968): View->render(NULL, NULL)
#2 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(200): Controller->render()
#3 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2022-04-22 17:50:15 Error: [MissingControllerException] Controller class LoginController could not be found.
Exception Attributes: array (
  'class' => 'LoginController',
  'plugin' => NULL,
)
Request URL: /mantenedor-usuarios/login
Stack Trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2022-04-22 18:08:57 Error: [MissingControllerException] Controller class AssetsController could not be found.
Exception Attributes: array (
  'class' => 'AssetsController',
  'plugin' => NULL,
)
Request URL: /mantenedor-usuarios/assets/brand/bootstrap-logo.svg
Stack Trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2022-04-22 18:14:11 Error: [MissingControllerException] Controller class AssetsController could not be found.
Exception Attributes: array (
  'class' => 'AssetsController',
  'plugin' => NULL,
)
Request URL: /mantenedor-usuarios/assets/brand/bootstrap-logo.svg
Stack Trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2022-04-22 18:14:15 Error: [MissingControllerException] Controller class AssetsController could not be found.
Exception Attributes: array (
  'class' => 'AssetsController',
  'plugin' => NULL,
)
Request URL: /mantenedor-usuarios/assets/brand/bootstrap-logo.svg
Stack Trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2022-04-22 18:14:31 Error: [MissingControllerException] Controller class AssetsController could not be found.
Exception Attributes: array (
  'class' => 'AssetsController',
  'plugin' => NULL,
)
Request URL: /mantenedor-usuarios/assets/brand/bootstrap-logo.svg
Stack Trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2022-04-22 18:15:01 Error: [MissingControllerException] Controller class AssetsController could not be found.
Exception Attributes: array (
  'class' => 'AssetsController',
  'plugin' => NULL,
)
Request URL: /mantenedor-usuarios/assets/brand/bootstrap-logo.svg
Stack Trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2022-04-22 18:15:36 Error: [MissingControllerException] Controller class AssetsController could not be found.
Exception Attributes: array (
  'class' => 'AssetsController',
  'plugin' => NULL,
)
Request URL: /mantenedor-usuarios/assets/brand/bootstrap-logo.svg
Stack Trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2022-04-22 18:16:10 Error: [MissingControllerException] Controller class AssetsController could not be found.
Exception Attributes: array (
  'class' => 'AssetsController',
  'plugin' => NULL,
)
Request URL: /mantenedor-usuarios/assets/brand/bootstrap-logo.svg
Stack Trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2022-04-22 18:16:39 Error: [MissingControllerException] Controller class AssetsController could not be found.
Exception Attributes: array (
  'class' => 'AssetsController',
  'plugin' => NULL,
)
Request URL: /mantenedor-usuarios/assets/brand/bootstrap-logo.svg
Stack Trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2022-04-22 18:19:32 Error: [MissingControllerException] Controller class AssetsController could not be found.
Exception Attributes: array (
  'class' => 'AssetsController',
  'plugin' => NULL,
)
Request URL: /mantenedor-usuarios/assets/brand/bootstrap-logo.svg
Stack Trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2022-04-22 18:39:01 Error: [Error] Call to undefined function Router()
Request URL: /mantenedor-usuarios/users
Stack Trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(971): include()
#1 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(933): View->_evaluate('C:\\xampp7_4_28\\...', Array)
#2 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(546): View->_render('C:\\xampp7_4_28\\...')
#3 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(481): View->renderLayout('', 'default')
#4 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Controller\Controller.php(968): View->render(NULL, 'default')
#5 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(200): Controller->render()
#6 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#7 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#8 {main}
2022-04-22 18:39:01 Error: Fatal Error (1): Uncaught Error: Call to undefined function Router() in C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\View\Layouts\default.ctp:31
Stack trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(971): include()
#1 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(933): View->_evaluate('C:\\xampp7_4_28\\...', Array)
#2 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(546): View->_render('C:\\xampp7_4_28\\...')
#3 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(481): View->renderLayout('<h2>Call to und...', 'default')
#4 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Controller\Controller.php(968): View->render('error500', 'default')
#5 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Error\ExceptionRenderer.php(292): Controller->render('error500')
#6 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Error\ExceptionRenderer.php(260): ExceptionRenderer->_outputMessage('error500')
#7 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Error\Exce in [C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\View\Layouts\default.ctp, line 31]
2022-04-22 18:39:01 Error: [FatalErrorException] Uncaught Error: Call to undefined function Router() in C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\View\Layouts\default.ctp:31
Stack trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(971): include()
#1 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(933): View->_evaluate('C:\\xampp7_4_28\\...', Array)
#2 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(546): View->_render('C:\\xampp7_4_28\\...')
#3 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(481): View->renderLayout('<h2>Call to und...', 'default')
#4 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Controller\Controller.php(968): View->render('error500', 'default')
#5 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Error\ExceptionRenderer.php(292): Controller->render('error500')
#6 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Error\ExceptionRenderer.php(260): ExceptionRenderer->_outputMessage('error500')
#7 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Error\Exce
Request URL: /mantenedor-usuarios/users
Stack Trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Error\ErrorHandler.php(212): ErrorHandler::handleFatalError(1, 'Uncaught Error:...', 'C:\\xampp7_4_28\\...', 31)
#1 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Core\App.php(970): ErrorHandler::handleError(1, 'Uncaught Error:...', 'C:\\xampp7_4_28\\...', 31, Array)
#2 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Core\App.php(943): App::_checkFatalError()
#3 [internal function]: App::shutdown()
#4 {main}
2022-04-22 18:39:16 Error: [Error] Call to undefined function Route()
Request URL: /mantenedor-usuarios/users
Stack Trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(971): include()
#1 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(933): View->_evaluate('C:\\xampp7_4_28\\...', Array)
#2 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(546): View->_render('C:\\xampp7_4_28\\...')
#3 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(481): View->renderLayout('', 'default')
#4 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Controller\Controller.php(968): View->render(NULL, 'default')
#5 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(200): Controller->render()
#6 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#7 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#8 {main}
2022-04-22 18:39:16 Error: Fatal Error (1): Uncaught Error: Call to undefined function Route() in C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\View\Layouts\default.ctp:31
Stack trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(971): include()
#1 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(933): View->_evaluate('C:\\xampp7_4_28\\...', Array)
#2 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(546): View->_render('C:\\xampp7_4_28\\...')
#3 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(481): View->renderLayout('<h2>Call to und...', 'default')
#4 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Controller\Controller.php(968): View->render('error500', 'default')
#5 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Error\ExceptionRenderer.php(292): Controller->render('error500')
#6 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Error\ExceptionRenderer.php(260): ExceptionRenderer->_outputMessage('error500')
#7 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Error\Excep in [C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\View\Layouts\default.ctp, line 31]
2022-04-22 18:39:16 Error: [FatalErrorException] Uncaught Error: Call to undefined function Route() in C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\View\Layouts\default.ctp:31
Stack trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(971): include()
#1 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(933): View->_evaluate('C:\\xampp7_4_28\\...', Array)
#2 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(546): View->_render('C:\\xampp7_4_28\\...')
#3 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(481): View->renderLayout('<h2>Call to und...', 'default')
#4 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Controller\Controller.php(968): View->render('error500', 'default')
#5 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Error\ExceptionRenderer.php(292): Controller->render('error500')
#6 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Error\ExceptionRenderer.php(260): ExceptionRenderer->_outputMessage('error500')
#7 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Error\Excep
Request URL: /mantenedor-usuarios/users
Stack Trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Error\ErrorHandler.php(212): ErrorHandler::handleFatalError(1, 'Uncaught Error:...', 'C:\\xampp7_4_28\\...', 31)
#1 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Core\App.php(970): ErrorHandler::handleError(1, 'Uncaught Error:...', 'C:\\xampp7_4_28\\...', 31, Array)
#2 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Core\App.php(943): App::_checkFatalError()
#3 [internal function]: App::shutdown()
#4 {main}
2022-04-22 19:24:16 Error: [MissingViewException] View file "Users\view.ctp" is missing.
Exception Attributes: array (
  'file' => 'Users\\view.ctp',
)
Request URL: /mantenedor-usuarios/Users/view/1
Stack Trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(470): View->_getViewFileName('Users\\view')
#1 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Controller\Controller.php(968): View->render(NULL, NULL)
#2 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(200): Controller->render()
#3 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2022-04-22 20:14:14 Error: [ParseError] syntax error, unexpected '}'
Request URL: /mantenedor-usuarios/Users
Stack Trace:
#0 [internal function]: App::load('UsersController')
#1 [internal function]: spl_autoload_call('UsersController')
#2 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(266): class_exists('UsersController')
#3 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(235): Dispatcher->_loadController(Object(CakeRequest))
#4 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(158): Dispatcher->_getController(Object(CakeRequest), Object(CakeResponse))
#5 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#6 {main}
2022-04-22 20:34:42 Error: [Error] Call to undefined method View::Elemen()
Request URL: /mantenedor-usuarios/Users/add
Stack Trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(971): include()
#1 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(933): View->_evaluate('C:\\xampp7_4_28\\...', Array)
#2 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(546): View->_render('C:\\xampp7_4_28\\...')
#3 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(481): View->renderLayout('<div class="car...', 'default')
#4 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Controller\Controller.php(968): View->render(NULL, 'default')
#5 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(200): Controller->render()
#6 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#7 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#8 {main}
2022-04-22 20:34:42 Error: Fatal Error (1): Uncaught Error: Call to undefined method View::Elemen() in C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\View\Layouts\default.ctp:50
Stack trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(971): include()
#1 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(933): View->_evaluate('C:\\xampp7_4_28\\...', Array)
#2 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(546): View->_render('C:\\xampp7_4_28\\...')
#3 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(481): View->renderLayout('<h2>Call to und...', 'default')
#4 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Controller\Controller.php(968): View->render('error500', 'default')
#5 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Error\ExceptionRenderer.php(292): Controller->render('error500')
#6 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Error\ExceptionRenderer.php(260): ExceptionRenderer->_outputMessage('error500')
#7 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Error\ in [C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\View\Layouts\default.ctp, line 50]
2022-04-22 20:34:42 Error: [FatalErrorException] Uncaught Error: Call to undefined method View::Elemen() in C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\View\Layouts\default.ctp:50
Stack trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(971): include()
#1 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(933): View->_evaluate('C:\\xampp7_4_28\\...', Array)
#2 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(546): View->_render('C:\\xampp7_4_28\\...')
#3 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\View\View.php(481): View->renderLayout('<h2>Call to und...', 'default')
#4 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Controller\Controller.php(968): View->render('error500', 'default')
#5 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Error\ExceptionRenderer.php(292): Controller->render('error500')
#6 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Error\ExceptionRenderer.php(260): ExceptionRenderer->_outputMessage('error500')
#7 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Error\
Request URL: /mantenedor-usuarios/Users/add
Stack Trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Error\ErrorHandler.php(212): ErrorHandler::handleFatalError(1, 'Uncaught Error:...', 'C:\\xampp7_4_28\\...', 50)
#1 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Core\App.php(970): ErrorHandler::handleError(1, 'Uncaught Error:...', 'C:\\xampp7_4_28\\...', 50, Array)
#2 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Core\App.php(943): App::_checkFatalError()
#3 [internal function]: App::shutdown()
#4 {main}
2022-04-22 20:54:39 Error: [MethodNotAllowedException] Method Not Allowed
Request URL: /mantenedor-usuarios/Users/desactivar/2/A
Stack Trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\Controller\UsersController.php(108): CakeRequest->allowMethod(Array)
#1 [internal function]: UsersController->desactivar('2', 'A')
#2 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#3 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#4 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#5 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#6 {main}
2022-04-22 20:54:58 Error: [MethodNotAllowedException] Method Not Allowed
Request URL: /mantenedor-usuarios/Users/desactivar/2/A
Stack Trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\Controller\UsersController.php(108): CakeRequest->allowMethod(Array)
#1 [internal function]: UsersController->desactivar('2', 'A')
#2 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#3 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#4 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#5 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#6 {main}
2022-04-22 20:55:12 Error: [MethodNotAllowedException] Method Not Allowed
Request URL: /mantenedor-usuarios/Users/desactivar/2/A
Stack Trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Network\CakeRequest.php(1095): CakeRequest->allowMethod(Array)
#1 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\Controller\UsersController.php(99): CakeRequest->onlyAllow(Array)
#2 [internal function]: UsersController->desactivar('2', 'A')
#3 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#4 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#5 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#6 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#7 {main}
2022-04-22 20:55:22 Error: [MethodNotAllowedException] Method Not Allowed
Request URL: /mantenedor-usuarios/Users/desactivar/2/A
Stack Trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\Controller\UsersController.php(106): CakeRequest->allowMethod(Array)
#1 [internal function]: UsersController->desactivar('2', 'A')
#2 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#3 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#4 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#5 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#6 {main}
2022-04-22 20:55:37 Error: [MethodNotAllowedException] Method Not Allowed
Request URL: /mantenedor-usuarios/Users/desactivar/2/A
Stack Trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\Controller\UsersController.php(106): CakeRequest->allowMethod(Array)
#1 [internal function]: UsersController->desactivar('2', 'A')
#2 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#3 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#4 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#5 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#6 {main}
2022-04-22 20:55:42 Error: [MethodNotAllowedException] Method Not Allowed
Request URL: /mantenedor-usuarios/Users/desactivar/2/A
Stack Trace:
#0 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\Controller\UsersController.php(106): CakeRequest->allowMethod(Array)
#1 [internal function]: UsersController->desactivar('2', 'A')
#2 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#3 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#4 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#5 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#6 {main}
2022-04-22 20:56:27 Error: [NotFoundException] Invalid Person
Request URL: /mantenedor-usuarios/Users/desactivar/2/A
Stack Trace:
#0 [internal function]: UsersController->desactivar('2', 'A')
#1 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#2 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp7_4_28\htdocs\mantenedor-usuarios\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 C:\xampp7_4_28\htdocs\mantenedor-usuarios\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
